install-scripts/install-ubuntu-dependencies.sh

```diff
@@ -40,12 +40,22 @@ sudo mv -v jwnl14-rc2/jwnl.jar /usr/local/share/java/
 rm -v jwnl14-rc2.zip && rmdir jwnl14-rc2
 sudo chmod -v 0644 /usr/local/share/java/jwnl.jar 
```

Comment:
Block of code above could be removed as library will be installed as maven dependency

Reply:
Updated.